packages/js/ai/src/hooks/useBackgroundRemoval.ts

```diff
@@ -0,0 +1,84 @@
+/**
+ * External dependencies
+ */
+import { useState } from 'react';
+import apiFetch from '@wordpress/api-fetch';
+
+export type BackgroundRemovalParams = {
+	imageFile: File;
+	returnPngImage?: boolean;
+	returnedImageSize?: string;
```

Comment:
It looks like this can only be one of a few discrete values, should we add that to the type? 
Ex `'preview' | 'medium' |'hd'`